test/webhook-handler/webhook-handler-test.js

```diff
@@ -115,4 +115,49 @@ describe('webhook handler tests', () => {
         })
     })
   })
+
+  describe('End to end tests', () => {
+    before(() => {
+      AWS_MOCK.mock('SSM', 'getParameter', { Value: 'secretkey' })
+      process.env.ALMA_SECRET_KEY_NAME = 'testkey'
+      process.env.AWS_DEFAULT_REGION = 'eu-west-2'
+    })
+
+    after(() => {
+      AWS_MOCK.restore('SSM')
+      delete process.env.ALMA_SECRET_KEY_NAME
+      delete process.env.AWS_DEFAULT_REGION
+    })
+
+    afterEach(() => {
+      AWS_MOCK.restore('SNS')
+    })
+
+    eventTopicData.forEach((data, topicName) => {
+      it('should call SNS publish with the event body & correct topic arn', (done) => {
```

Comment:
Can you use string interpolation from the topic name so the test name reads correctly and if a specific one breaks its easy to understand which on this is?

Reply:
This should be fixed in https://github.com/lulibrary/alma-webhook-handler/pull/12/commits/55d6ca7444e0a04e2c564d2deac704b93cefbaab